### PR TITLE
Changelog for Rust 1.75 :christmas_tree:

### PR DESCRIPTION
**Pull request by xFrednet**
Roses are red,
Violets are blue,
Another year of development,
And a strong team, too

---

### The cat of this release:

The cats of this Clippy release

<sub>The cat for the next release can be nominated in the comments</sub>

---

changelog: none

### CODE REVIEW

**Comment by ThomasMiz** [comment]
I nominate Nina
image

**Comment by xFrednet** [comment]
@ThomasMiz Cute! Nomination accepted and thank you! (Also, welcome to the weirdness the changelog PRs have become :D)

**Comment by Baptistemontan** [comment]
I nominate Berlioz image

**Comment by blyxyas** [comment]
Wait! More cats incoming, we should wait until all submissions are taken, and **then** choose one. Democracy and that kinda thing

**Comment by xFrednet** [comment]
Usually, it was always that the first one won because we always only had one nomination. That path isn't implemented yet :sob: 

```rs
fn choose_nomination(noms: &[Image]) -> Image {
    match noms {
        [img] -> img,
        _ -> todo!(),
    }
}
```

**Comment by SLASHLogin** [comment]
I nominate Victoria the Ball
V-Ball

**Comment by syndek** [comment]
Just stopping by to nominate Luigi :-)
D3104E93-6AD9-4C98-952D-E35F8000532F_4_5005_c

**Comment by PartiallyTyped** [comment]
This lovable fuzzball that can't seem to let go is Misty 💜

image

**Comment by flip1995** [comment]
> That path isn't implemented yet

If only we had `#![warn(clippy::wildcard_enum_match_arm, clippy::todo)]` enabled!

How should we ever choose one of those adorable cats? I had to leave a ❤️ for all of them 😿

**Comment by flip1995** [comment]
> Is it cheating if I share multiple pics?

More pics are appreciated, but it won't get you extra credits when choosing the cat

**Comment by flip1995** [comment]
@bors r+

Thanks, great poem as always! 

With the merge of this PR I'd say we close the nomination for the next changelog. Keep your pics ready for the next round in 6 weeks.

Making up rules for choosing a cat, I'll leave to the experts @blyxyas @xFrednet 🙇

**Comment by gautam8404** [comment]
These strays i found, my camera quality sucks but they all are adorable

**Comment by blyxyas** [comment]
Oh! Really getting in those last seconds before bors merges this!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,68 @@ document.
...
+* Moved [`missing_enforced_import_renames`] to `style` (Now warn-by-default)
+  [#11539](https://github.com/rust-lang/rust-clippy/pull/11539)
+* Moved [`needless_raw_string_hashes`] to `pedantic` (Now allow-by-default)
+  [#11415](https://github.com/rust-lang/rust-clippy/pull/11415)
+* Moved [`needless_pass_by_ref_mut`] to `nursery` (Now allow-by-default)
+  [#11596](https://github.com/rust-lang/rust-clippy/pull/11596)
+
+### Enhancements
+
+* [`declare_interior_mutable_const`] and [`borrow_interior_mutable_const`]: Now follow check the
```
Comment (llogiq) [inline CHANGELOG.md:48]: This looks weird. From my perspective, they either follow or check the value, but not both.

Comment (xFrednet) [reply CHANGELOG.md:48]: Woops, obviously I had to have a typo somewhere, will fix it!

**Comment by valyagolev** [comment]
Istar
PXL_20231219_133641859 MP~2

**Comment by llogiq** [comment]
I'd like to nominate Tom a.k.a. "Maunzer" (= he who meows in German).

**Comment by daniel-g-gagnon** [comment]
I’d like to nominate Cabrel in his cute halloween costume.

Cabrel

**Comment by AestheticSkye** [comment]
I’d like to nominate Baked Beans, and his little vampire fangs
Untitled

**Comment by robot-head** [comment]
I nominate SPACECAT
PXL_20220602_184710717

**Comment by PartiallyTyped** [comment]
I can't get enough of this thread ❤️

**Comment by robot-head** [comment]
> I can't get enough of this thread ❤️

image

**Comment by larajones** [comment]
I have three and cannot show favoritism so I have three nominations:

Gabriel
IMG_1218

Castiel
IMG_1788

Aiden
IMG_1919

**Comment by liuspatt** [comment]
Blacy
IMG_2590

**Comment by ThomasMiz** [comment]
> > I can't get enough of this thread ❤️
> 
> image

Hahaha I love this! Just because of that, here's Nina playing with a ribbon:
image

**Comment by lensvol** [comment]
I nominate Zola, Queen of the Reading Chair:

image

**Comment by Lisalotte** [comment]
I nominate Floof

**Comment by Lisalotte** [comment]
And also, Sammy

**Comment by harr1424** [comment]
IMG_2775

Pinto!

**Comment by DawidPietrykowski** [comment]
I nominate Rudolf ❤️

**Comment by PortableNavi** [comment]
I nominate Fleur

**Comment by blyxyas** [comment]
People, please, keep your cat photos for the next release! You can post them on just 6 weeks! It's just too much, we cannot handle so much kitty-ness :sob:

Maybe we'll port this tradition to Discord, stay tuned to this issue (but we'll probably give news through [our Reddit](https://github.com/r/rust))!

**Comment by blyxyas** [comment]
Talking with Fred, we've decided that we'll hold this PR for 24 hours open for nominations. I'll re-open it until Dec. 27, 4:10 PM GMT, then we'll reclose (maybe someone didn't have their opportunity, we closed a little bit prematurely)

**Comment by Forest828** [comment]
Here is Lisa the void

**Comment by MasterAwesome** [comment]
Here's Hestia, she didn't like the paparazzi :p

**Pull request merged**